scripts/deploy.ts

```diff
@@ -67,6 +67,15 @@ async function main() {
     address_minter = process.env.AVAX_FIREBLOCKS_MINTER;
     address_rescuer = process.env.AVAX_FIREBLOCKS_RESCUER;
     address_burner = process.env.AVAX_FIREBLOCKS_BURNER;
+  }else if (network.name =="gaiaxDirect"){
+    address_proxyOwner = process.env.GAIAX_TESTNET_DEVELOPER_KEY;
```

Comment:
Well now this is here DEVELOPER_KEY but in example it's DEVELOPER_ADDRESS

Reply:
named it address.